frontend/src/components/housing/maps/Map.vue

```diff
@@ -44,6 +46,26 @@ async function initMap() {
   });
 }
 
+function disableInteraction() {
+  map.dragging.disable();
+  map.touchZoom.disable();
+  map.doubleClickZoom.disable();
+  map.scrollWheelZoom.disable();
+  map.boxZoom.disable();
+  map.keyboard.disable();
+  if (map.tap) map.tap.disable();
+}
+
+function enableInteraction() {
+  map.dragging.enable();
+  map.touchZoom.enable();
+  map.doubleClickZoom.enable();
+  map.scrollWheelZoom.enable();
+  map.boxZoom.enable();
+  map.keyboard.enable();
+  if (map.tap) map.tap.enable();
+}
```

Comment:
Optional Style: With effectively two identical lists of items being permuted on, it might be worth considering jamming these map attributes into a defined array to reduce duplication and future accidental copy/pasta divergences:
```ts
const interactionElements = [map.dragging, map.touchZoom, map.doubleClickZoom, map.scrollWheelZoom, map.boxZoom, map.keyboard];

function disableInteraction() {
  interactionElements.forEach(elem => elem.disable());
  if (map.tap) map.tap.disable();
}

function enableInteraction() {
  interactionElements.forEach(elem => elem.enable());
  if (map.tap) map.tap.enable();
}
```

Reply:
There are some Typescript issues with the map type and its constituent functions that make this cleaner approach not as useful. Will look into this at another time.